src/test/java/uk/gov/verifiablelog/merkletree/MerkleTreeMockTests.java

```diff
@@ -82,6 +83,69 @@ public void currentRoot_should_retrieveAndUseMemoizationStoreHashes() {
         assertThat(bytesToString(rootHash), is(bytesToString(expectedRootHash)));
     }
 
+    @Test
+    public void pathToRootAtSnapshot_should_retrieveAndUseMemoizationStoreHashes() {
+        /**
+         *  Tree and memoization expectations for audit proof 4, 8
+         *  M - memoized
```

Comment:
does `memoized` mean "memoized before the test" or "memoized by the code under test"?